src/lib/y2network/config.rb

```diff
@@ -79,7 +80,8 @@ def copy
     #
     # @return [Boolean] true if both configurations are equal; false otherwise
     def ==(other)
-      source == other.source && (interfaces - other.interfaces).empty? &&
+      source == other.source &&
+        ((interfaces - other.interfaces) | (other.interfaces - interfaces)).empty? &&
```

Comment:
ok, this will work now, even I am not sure if it is more readable then `interfaces.sort.uniq == other.interfaces.sort.uniq`

Reply:
I did not use `#sort` because I am not sure about the sorting logic... by name? by type? Does it really make sense to sort devices?

Reply:
I think by name should be fine. Just to ensure stable ordering for comparison.